app/src/main/java/com/example/nvdovin/weatherapp/domain/utils/DateConvertor.java

```diff
@@ -0,0 +1,29 @@
+package com.example.nvdovin.weatherapp.domain.utils;
+
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class DateConvertor {
+
+    public static final int MILI_VALUE = 1000;
```

Comment:
The constant needed to convert from Milliseconds to seconds, had to extract 1000 in to constant